data/human/paradise world jobs.txt

```diff
@@ -393,6 +393,28 @@ mission "Transport high-class tourists"
 		payment 25000 250
 
 
+mission "Transport high-class tourist group"
+	description "Bring <bunks> outrageously wealthy tourists on a fabulous journey to the wild and exotic world of <destination>. They'll only consider thinking about your ship if you have the 'appropriate facilities' for a comfortable journey. Payment is <payment>."
+	job
+	repeat
+	passengers 15 5 .15
+	to offer
+		random < 30
+		random < "passenger space" - 35
```

Comment:
Intentional double random? This mission doesn't seem lucrative enough to me to justify it.